### ISSUE TITLE
Adapt Accessibility tests: close Drawer button

### ISSUE THREAD

**Issue by shauncg** [adaptlearning/adapt_framework#401]
The screenreader text for the [X] button on the Drawer is simply "Close link". This should be "Close Drawer link" (or otherwise consistent with the open Drawer button text - see issue ticket #400 ).
